docs/services/frame-system/_index.md

```diff
@@ -207,11 +207,125 @@ The resulting tree of reference frames looks like:
+- `destination` (str): The name of the reference frame to transform the given pose to.
+- `additional_transforms` (Optional[List[[Transform](https://python.viam.dev/autoapi/viam/proto/common/index.html#viam.proto.common.Transform)]]): A list of [additional transforms](#supplemental-transforms).
+
+**Returns:**
+
+- `PoseInFrame` [(PoseInFrame)](https://python.viam.dev/autoapi/viam/proto/common/index.html#viam.proto.common.PoseInFrame): Transformed pose in destination reference frame.
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/robot/client/index.html#viam.robot.client.RobotClient.transform_pose).
+
+```python {class="line-numbers linkable-line-numbers"}
+# Transform a pose into the frame of myArm
+first_pose = Pose(x=0.0, y=0.0, z=0.0, o_x=0.0, o_y=0.0, o_z=1.0, theta=0.0)
+first_pif = PoseInFrame(reference_frame="world", pose=first_pose)
+transformed_pif = await robot.transform_pose(first_pif, "myArm")
+print("Position: (x:", transformed_pif.pose.x, ", y:", transformed_pif.pose.y, ", z:", transformed_pif.pose.z, ")")
+print("Orientation: (o_x:", transformed_pif.pose.o_x,
+      ", o_y:", transformed_pif.pose.o_y,
+      ", o_z:",transformed_pif.pose.o_z,
+      ", theta:", transformed_pif.pose.theta, ")")
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- `ctx` [(`Context`)](https://pkg.go.dev/context): A Context carries a deadline, a cancellation signal, and other values across API boundaries.
+- `pose` (Optional[[PoseInFrame](https://pkg.go.dev/go.viam.com/rdk/referenceframe#PoseInFrame)]): The pose that should be transformed.
+- `dst` (string): The name of the reference frame to transform the given pose to.
+- `additionalTransforms` (Optional[[LinkInFrame](https://pkg.go.dev/go.viam.com/rdk/referenceframe#LinkInFrame)]): A list of [additional transforms](#supplemental-transforms).
+
+**Returns:**
+
+- `error` [(`error`)](https://pkg.go.dev/builtin#error): An error, if one occurred.
+- `PoseInFrame` [(referenceframe.PoseInFrame)](https://pkg.go.dev/go.viam.com/rdk/referenceframe#PoseInFrame): Transformed pose in destination reference frame.
+
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/robot).
+
+```go {class="line-numbers linkable-line-numbers"}
+// Define a Pose coincident with the world reference frame
+firstPose := spatialmath.NewPoseFromPoint(r3.Vector{X: 0.0, Y: 0.0, Z: 0.0})
+
+// Establish the world as the reference for firstPose
+firstPoseInFrame := referenceframe.NewPoseInFrame(referenceframe.World, firstPose)
+
+// Calculate firstPoseInFrame from the perspective of the origin frame of myArm
+transformedPoseInFrame, err := robot.TransformPose(ctx, firstPoseInFrame, "myArm", nil)
+fmt.Println("Transformed Position:", transformedPoseInFrame.Pose().Point())
+fmt.Println("Transformed Orientation:", transformedPoseInFrame.Pose().Orientation())
+```
+
+{{% /tab %}}
+{{< /tabs >}}
 
 ## Supplemental Transforms
```

Comment:
Unfortunately, above the parameter is called `additionalTransforms` - so I can say additional supplemental transforms but to really close the loop would require rdk changes I think.

Reply:
Can we rename this section to "Additional Transforms"?

Reply:
```suggestion
## Additional Transforms
```

Reply:
@biotinker The motion service uses `supplemental_transforms` as the parameter name - I can change it in the rest of the text but not in the example. I guess we can have that be the only inconsistency but I thought it might be better to then keep it as supplemental for the paragraph

Reply:
Eugh, we're inconsistent with "additional" vs "supplemental" naming in motion vs FS? Did not realize that :(

@spieswl heads up

Reply:
We ok to merge? @biotinker